### PR TITLE
docs(apps/docs): add favicon and social image

### PR DESCRIPTION
**Pull request by r3nanp**

### CODE REVIEW

**Comment by sannajammeh** [comment]
Great stuff! Thanks for the PR. I see we need the open graph data for all pages, so removing the if check on the asPath would be preferred here. Set the title when asPath is === `/` to whatever you see fit, and otherwise leave it as it is. The rest of the openGraph is needed on all routes.

File: apps/docs/theme.config.tsx
Comment (sannajammeh) [inline apps/docs/theme.config.tsx:1]: Remove line 9 `if (asPath !== "/") {`

And replace line 23 `titleTemplate: "%s - tRPC SWR",` with `titleTemplate: asPath !== "/" ? "tRPC SWR" : "%s - tRPC SWR",`

**Review by sannajammeh — CHANGES_REQUESTED** [review]
[apps/docs/theme.config.tsx](https://github.com/sannajammeh/trpc-swr/pull/45/files#diff-3a56172e097b6bd1ba518b93cd20c5af7f3a731ab8683c6f8b8e84044b695ad3)

File: apps/docs/theme.config.tsx
Comment (r3nanp) [reply apps/docs/theme.config.tsx:1]: Thanks for the heads up!

**Review by sannajammeh — APPROVED** [review]
Great stuff!

**Comment by r3nanp** [comment]
A small problem in this GitHub actions job:
https://github.com/sannajammeh/trpc-swr/actions/runs/4997532792/jobs/8952024363?pr=45

Yep its a slight permissions issue due to forks. I'll look into it

**Pull request merged**